openstack/containerinfra/v1/clustertemplates/testing/fixtures.go

```diff
@@ -0,0 +1,197 @@
+package testing
+
+import (
+	"fmt"
+	"net/http"
+	"testing"
+	"time"
+
+	"github.com/gophercloud/gophercloud"
+	"github.com/gophercloud/gophercloud/openstack/containerinfra/v1/clustertemplates"
+	th "github.com/gophercloud/gophercloud/testhelper"
+	fake "github.com/gophercloud/gophercloud/testhelper/client"
+)
+
+const ClusterTemplateResponse = `
```

Comment:
These fixtures look to be pulled from the old `magnum` branch and are outdated (note how the link includes `baymodels`). Please generate fresh fixtures from a working Magnum environment.